test/Microsoft.Health.Fhir.Tests.E2E/Common/FhirClient.cs

```diff
@@ -293,19 +294,31 @@ private async Task<string> GetBearerToken(TestApplication clientApplication, Tes
 
             var tokenJson = JObject.Parse(await tokenResponse.Content.ReadAsStringAsync());
 
-            var bearerToken = tokenJson["access_token"].Value<string>();
-            return bearerToken;
+            if (clientApplication.Equals(TestApplications.InvalidClient))
```

Comment:
nit: We could return this case immediately and not attempt to even get the json value.

Reply:
Done